Gemfile

```diff
@@ -0,0 +1,55 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.4.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
```

Comment:
Remove commented out code.

Reply:
Removed comments from gemfile is now done.